internal/pkg/cli/override.go

```diff
@@ -172,16 +173,20 @@ func (o *overrideOpts) askResourcesToOverride() error {
 		return nil
 	}
 
+	o.spinner.Start("Generating CloudFormation template to select resources")
```

Comment:
nit: would users be slightly confused when they saw this? (disclaimer: I am not an english expert 😂 )
```suggestion
	o.spinner.Start("Generating CloudFormation template to select resources from")
```